src/components/ClusterOverview/Inventory/Inventory.js

```diff
@@ -1,80 +1,66 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Col, Icon } from 'patternfly-react';
 
-import { getNodeErrorStatuses } from './nodeStatus';
 import {
   DashboardCard,
   DashboardCardBody,
   DashboardCardHeader,
   DashboardCardTitle,
   DashboardCardTitleHelp,
 } from '../../Dashboard/DashboardCard';
-import { ClusterOverviewContextGenericConsumer } from '../ClusterOverviewContext';
+import { ClusterOverviewContext } from '../ClusterOverviewContext';
+import { mapNodesToProps, mapPodsToProps, mapPvcsToProps, mapVmsToProps } from './utils';
+import { InventoryRow } from './InventoryRow';
 
-const InventoryItemStatus = ({ item }) => {
-  let ok = true;
-  if (item.kind === 'Node') {
-    ok = !item.data.some(i => getNodeErrorStatuses(i).length > 0);
-  }
-  return ok ? (
-    <div className="kubevirt-inventory__item-status">
-      <Icon type="fa" name="check-circle" size="2x" className="kubevirt-inventory__item-status-icon" />
-      <span>{item.data.length}</span>
-    </div>
-  ) : (
-    <div>
-      <span className="glyphicon glyphicon-remove-circle" />
-    </div>
-  );
-};
+const InventoryBody = ({ nodes, pods, vms, vmis, pvcs, migrations }) => (
+  <React.Fragment>
+    <InventoryRow title="Hosts" {...mapNodesToProps(nodes)} />
```

Comment:
Nodes and hosts are two different things, AFAIK

Reply:
Yes, they are. I changed it to `Nodes`.

Nevertheless, I don't what is the resolution about this issue. If both nodes and hosts will be present in this card.